theano/sandbox/gpuarray/tests/test_abstractconv.py

```diff
@@ -1,7 +1,10 @@
 from nose.plugins.skip import SkipTest
 
+import numpy
+from pygpu import gpuarray
```

Comment:
The pygpu import should be after line 10, otherwise, it make tests crash when pygpu isn't there like on travis.

The other travis errors are fixed by https://github.com/Theano/Theano/pull/4176